### ISSUE TITLE
Have you developed face mask detection using tensorflow ? 

### ISSUE THREAD

**Issue by ganeshchenniah** [Rajatkalsotra/Face-Recognition-Flutter#4]

**Comment by Rajatkalsotra** [comment]
@ganeshchenniah I used Firebase ml vision package for that

**Comment by ganeshchenniah** [comment]
Hi ,
Yes , you used mlkit for face detection , is it possible to include mask
detection ??.

**Comment by Rajatkalsotra** [comment]
@ganeshchenniah For mask detection you have to train model for that specific purpose and then you can pass detected face to it.

**Comment by ganeshchenniah** [comment]
Hi ,
 I have a mask detection dataset , how did you train the model for 112*112
images , can u ou give pointer to create trained model ?

**Comment by Rajatkalsotra** [comment]
@ganeshchenniah I've used pre-trained Mobilefacenet model. 
For your problem:
- Detect faces using opencv ,then crop and resize faces (images) to the same size( 112*112)
- Train keras model (CNN) classifier to detect mask and save it (h5).
- Convert trained model into tflite model. 
- Then use that tflite model in the flutter app.
